cmd/templates.go

```diff
@@ -21,3 +27,44 @@ Flags:
 {{.LocalFlags.FlagUsages | trimTrailingWhitespaces}}
 `
 }
+
+func WrappedRequiredFlagUsages(cmd *cobra.Command) string {
+	nonRequestParamsFlags := pflag.NewFlagSet("request", pflag.ExitOnError)
+
+	cmd.LocalFlags().VisitAll(func(flag *pflag.Flag) {
+		if _, ok := flag.Annotations["required"]; ok {
+			nonRequestParamsFlags.AddFlag(flag)
+		}
+	})
+
+	return nonRequestParamsFlags.FlagUsagesWrapped(getTerminalWidth())
+}
+
+func WrappedOptionalFlagUsages(cmd *cobra.Command) string {
+	nonRequestParamsFlags := pflag.NewFlagSet("request", pflag.ExitOnError)
+
+	cmd.LocalFlags().VisitAll(func(flag *pflag.Flag) {
+		_, ok := flag.Annotations["required"]
+		if !ok && flag.Name != "help" {
+			nonRequestParamsFlags.AddFlag(flag)
+		}
+	})
+
+	return nonRequestParamsFlags.FlagUsagesWrapped(getTerminalWidth())
+}
+
+func getTerminalWidth() int {
+	var width int
+
+	width, _, err := term.GetSize(0)
+	if err != nil {
+		width = 80
+	}
+
+	return width
+}
+
+func init() {
+	cobra.AddTemplateFunc("WrappedRequiredFlagUsages", WrappedRequiredFlagUsages)
```

Comment:
Could you move these to [`Execute()`](https://github.com/launchdarkly/ldcli/blob/main/cmd/root.go#L190) so we don't use `init()`?

Reply:
oh yup!